cves/CVE-2018-9841.yml

```diff
@@ -90,7 +98,7 @@ fixes_vcc_instructions: |
   find interesting in there.
 fixes:
 - commit: 35eeff30caf34df835206f1c12bcf4b7c2bd6758
-  note: ''
+  note: Fixes out of array access
 - commit: ''
   note: ''
 vccs:
```

Comment:
Did you actually check the VCC?

Reply:
Yes, the vcc commit contains a method that calls strcpy without checking the size of the str (from input) being copied. Is this right?

Reply:
You don't need to add a note, unless the code in the VCC isn't clear -- I just wanted to make sure you actually looked at the VCC and verified that it was correct.

Reply:
Okay sounds good.